### PR TITLE
Made ERC721 contracts upgradeable & tests improvements

### PR DESCRIPTION
**Pull request by dovgopoly**

### CODE REVIEW

File: contracts/gov/ERC721/ERC721Multiplier.sol [hunk 1]
```diff
@@ -20,7 +20,14 @@ contract ERC721Multiplier is IERC721Multiplier, ERC721Enumerable, Ownable {
     event Minted(address to, uint256 tokenId, uint256 multiplier, uint256 duration);
     event Locked(address from, uint256 tokenId, uint256 multiplier, uint256 duration);
 
-    constructor(string memory name, string memory symbol) ERC721(name, symbol) {}
+    function __ERC721Multiplier_init(
+        string memory name,
```
Comment (Arvolear) [inline contracts/gov/ERC721/ERC721Multiplier.sol:24]: calldata?

File: contracts/gov/ERC721/ERC721Power.sol [hunk 2]
```diff
@@ -43,15 +43,19 @@ contract ERC721Power is IERC721Power, ERC721Enumerable, Ownable {
         _;
     }
 
-    constructor(
+    function __ERC721Power_init(
         string memory name,
```
Comment (Arvolear) [inline contracts/gov/ERC721/ERC721Power.sol:47]: calldata?

Comment (dovgopoly) [reply contracts/gov/ERC721/ERC721Power.sol]: done

**Review by Arvolear — APPROVED** [review]
Ok

**Pull request merged**